bftengine/src/bftengine/messages/MsgsCertificate.hpp

```diff
@@ -288,7 +288,7 @@ bool MsgsCertificate<T, SelfTrust, SelfIsRequired, KeepAllMsgs, ExternalFunc>::i
 
 template <typename T, bool SelfTrust, bool SelfIsRequired, bool KeepAllMsgs, typename ExternalFunc>
 bool MsgsCertificate<T, SelfTrust, SelfIsRequired, KeepAllMsgs, ExternalFunc>::isInconsistent() const {
-  return (numOfClasses >= maxFails + 1);
+  return (numOfClasses > maxFails + 1);
```

Comment:
which test case discovered this problem?

@yuliasherman this looks like a missed corner case - what do you think?

Reply:
- I read the code, and understood the number doesn't fit the protocol.
  Also checked with Guy to be sure, he confirmed that indeed this is incorrect.

Reply:
Great job! Can you also write a unit test that specifically reproduces this problem?

Reply:
As discussed offline, the test already exists, and will be mentioned in the PR description.